cucumber-rails.gemspec

```diff
@@ -26,9 +26,9 @@ Gem::Specification.new do |s|
   s.add_development_dependency('bundler', '>= 1.3.5')
   s.add_development_dependency('selenium-webdriver', '>= 3.4.1')
   s.add_development_dependency('database_cleaner', '>= 1.0.0')
-  s.add_development_dependency('factory_girl', '>= 3.2')
+  s.add_development_dependency('factory_bot', '>= 3.2')
   s.add_development_dependency('rake', '>= 0.9.2.2')
-  s.add_development_dependency('rspec', '~> 3.0')
+  s.add_development_dependency('rspec', '~> 3')
```

Comment:
This should become `~> 3.0` again to avoid allowing rspec 4 automatically when it arrives.

Reply:
Woops. I'll fix that now - thanks @mvz!